src/app/app.module.ts

```diff
@@ -67,7 +67,10 @@ import { OwnershipComponent } from './ownership/ownership.component';
 import { CurrencyDefaultPipe } from './pipes/currency-default-pipe';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
remove ng-boostrap from the project and use ModalModule from ngx-bootstrap instead.